models/note.go

```diff
@@ -10,33 +10,34 @@ type NoteId int64
 type NoteType int
 
 const (
-	MARGINALIA NoteType = iota
+	UNCATEGORIZED NoteType = iota
+	MARGINALIA
 	META
 	QUESTIONS
 	PREDICTIONS
 )
 
 var noteTypeStrings = [...]string{
-	"Marginalia",
-	"Meta",
-	"Questions",
-	"Predictions",
+	"uncategorized",
+	"marginalia",
```

Comment:
I lower all these strings so they would match 1:1 with the strings in the Databases enum. Since when i compared them in the past the DB was case sensitive for enum matching.